.cirrus.yml

```
@@ -1,104 +1,152 @@
-task:
-  auto_cancellation: true
+# default container
+container:
+  image: python:3.8
 
-  name: Black Formatting
-  container:
-      image: python:3.6
 
+black_task:
```

```suggestion
task:
```

Avoid doing this so the parser has less work to do

---

seems strange... in valid yaml, mapping keys are unique (even if it still works)... what is the "cost" of the parser having less work to do?  are we talking potential bugs?  or a tiny bit of time?

---

not to mention that's how cirrus encourages it [in their own docs](https://cirrus-ci.org/guide/writing-tasks/)... would need to see the reference for the downside of having unique keys like this

---

Never mind